pkg/querier/tenantfederation/merge_queryable.go

```diff
@@ -118,17 +119,27 @@ func NewMergeQueryable(idLabelName string, callbacks MergeQueryableCallbacks, re
 			Help:    "Number of tenants queried for a single standard query.",
 			Buckets: []float64{1, 2, 4, 8, 16, 32},
 		}),
+
+		// Experimental: Observe time to kick off upstream query jobs as a native histogram
+		upstreamQueryWaitDuration: promauto.With(reg).NewHistogram(prometheus.HistogramOpts{
+			Name:                            "mimir_querier_federation_upstream_query_wait_duration_seconds",
```

Comment:
I might be wrong, but I don't think we have any metrics with the `mimir_` prefix - even for new metrics, we've kept the `cortex_` prefix.